core/kazoo_ast/src/cb_api_endpoints.erl

```diff
@@ -253,13 +253,12 @@ process_schema(Filename, Definitions) ->
     JObj = kz_json:expand(
              kz_json:from_list(
                [case lists:last(Path) =:= <<"$ref">> of
-                    false -> KV;
-                    true -> {Path, maybe_fix_ref(V)}
+                    'false' -> KV;
+                    'true' -> {Path, maybe_fix_ref(V)}
                 end
                 || {Path, V}=KV <- kz_json:to_proplist(kz_json:flatten(JObj0)),
                    not lists:member(<<"patternProperties">>, Path),
-                   not lists:member(<<"kazoo-validation">>, Path),
-                   not lists:member(<<"oneOf">>, Path)
+                   not lists:member(<<"kazoo-validation">>, Path)
```

Comment:
since you're changing this, can you add `not lists:member(<<"kazoo-", _/binary>>, Path)` instead ?

Reply:
hehe, that's not exactly valid but I get what you're asking :)